### PR TITLE
[Doc improvement] Document recommended length of labels

### PR DESCRIPTION
**Pull request by samantharamon**
Documents that "tabs.label", "groups.label", and "controls.label" should be limited to 16 characters.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/64eac93ef9a0d69e1afe285baccdc261e8b5cefd/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 83 | 131 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/54de7e52-3d3c-4b3b-ae37-c7370375480d) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/64eac93ef9a0d69e1afe285baccdc261e8b5cefd/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema.md) | 86 | 115 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3870ec47-5609-4151-9c60-284fd135c820) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c3d2f93fae4f54d601b9a586d7185de28425d7dc/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 84 | 126 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0432f2fb-2f40-4815-8b56-97bb673c56c5) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c3d2f93fae4f54d601b9a586d7185de28425d7dc/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema.md) | 86 | 109 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9d1455d6-1666-485a-8a44-527fc85c62e5) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by samantharamon** [comment]
@Rick-Kirkham, when you get a chance, could you please review this PR? Thanks.

**Review by Rick-Kirkham — DISMISSED** [review]

File: msteams-platform/resources/schema/manifest-schema-dev-preview.md [hunk 1]
```diff
@@ -1017,14 +1017,14 @@ The `extensions.ribbons` property provides the ability to add [add-in commands](
...
-|`tabs.label`| String | 64 characters | | Specifies the text displayed for the tab.|
+|`tabs.builtinTabId`| String | 64 characters | | Specifies the ID of a built-in Office ribbon tab. The possible values vary by Office host application. Outlook add-ins are only supported and the allowed value for Outlook is "TabDefault". The default tab depends on where the Outlook add-in is surfaced, as determined in the "extensions.ribbons.contexts" property. In the main Outlook window, it's the **Home** tab, in a message window, it's the **Message** tab, and in a meeting window, it's the **Meeting** tab. |
+|`tabs.label`| String | 64 characters | | Specifies the text displayed for the tab.<br>**Recommendation**: Although the maximum length is 64 characters, to ensure that the tab fits correctly in the ribbon, it's a good practice to limit the label to 16 characters.|
```
Comment (v-npaladugu) [inline msteams-platform/resources/schema/manifest-schema-dev-preview.md:1021]:
```suggestion
|`tabs.label`| String | 64 characters | | Specifies the text displayed for the tab. Despite the maximum length being 64 characters, for the tab to align correctly in the ribbon, we recommend you limit the label to 16 characters.|
```

File: msteams-platform/resources/schema/manifest-schema-dev-preview.md [hunk 2]
```diff
@@ -1017,14 +1017,14 @@ The `extensions.ribbons` property provides the ability to add [add-in commands](
...
 |`tabs.position.align`| String enum | | ✔️ | Defines the alignment of custom tab relative to the specified built-in tab. <br>Supported values: `after`, `before`|
 |`tabs.groups`| Array | | | Defines groups of controls on a ribbon tab on a non-mobile device. For mobile devices, see `tabs.customMobileRibbonGroups` below.|
 |`tabs.groups.id`| String |64 characters | | Specifies the ID for the tab group within the app. It must be different from any built-in group ID in the Microsoft 365 application and any other custom group.|
-|`tabs.groups.label`| String | 64 characters | | Specifies the text displayed for the group. |
+|`tabs.groups.label`| String | 64 characters | | Specifies the text displayed for the group.<br>**Recommendation**: Although the maximum length is 64 characters, to ensure that the group of controls fits correctly in the ribbon, it's a good practice to limit the label to 16 characters.|
```
Comment (v-npaladugu) [inline msteams-platform/resources/schema/manifest-schema-dev-preview.md:1027]:
```suggestion
|`tabs.groups.label`| String | 64 characters | | Specifies the text displayed for the group. Despite the maximum length being 64 characters, for the group of controls to align correctly in the ribbon, we recommend you limit the label to 16 characters.|
```

File: msteams-platform/resources/schema/manifest-schema-dev-preview.md [hunk 3]
```diff
@@ -1045,14 +1045,14 @@ The `extensions.ribbons` property provides the ability to add [add-in commands](
...
 |`tabs.groups.controls.type`| String | | ✔️ | Defines the control type. <br>Supported values: `button`, `menu`|
 |`tabs.groups.controls.builtinControlId`| String | 64 characters | ✔️ | Specifies the ID of an existing Microsoft 365 control. For more information, see [find the IDs of controls and control groups](/office/dev/add-ins/design/built-in-button-integration#find-the-ids-of-controls-and-control-groups).|
-|`tabs.groups.controls.label`| String | 64 characters | ✔️ | Specifies the text displayed for the control.|
+|`tabs.groups.controls.label`| String | 64 characters | ✔️ | Specifies the text displayed for the control.<br>**Recommendation**: Although the maximum length is 64 characters, to ensure that the control fits correctly in the ribbon, it's a good practice to limit the label to 16 characters.|
```
Comment (v-npaladugu) [inline msteams-platform/resources/schema/manifest-schema-dev-preview.md:1048]:
```suggestion
|`tabs.groups.controls.label`| String | 64 characters | ✔️ | Specifies the text displayed for the control. Despite the maximum length being 64 characters, for the control to align correctly in the ribbon, we recommend you limit the label to 16 characters. |
```

File: msteams-platform/resources/schema/manifest-schema.md [hunk 4]
```diff
@@ -1050,14 +1050,14 @@ The `extensions.ribbons` property provides the ability to add [add-in commands](
...
-|`tabs.label`| String | 64 characters | | Specifies the text displayed for the tab.|
+|`tabs.builtinTabId`| String | 64 characters | | Specifies the ID of a built-in Office ribbon tab. The possible values vary by Office host application. Outlook add-ins are only supported and the allowed value for Outlook is "TabDefault". The default tab depends on where the Outlook add-in is surfaced, as determined in the "extensions.ribbons.contexts" property. In the main Outlook window, it's the **Home** tab, in a message window, it's the **Message** tab, and in a meeting window, it's the **Meeting** tab. |
+|`tabs.label`| String | 64 characters | | Specifies the text displayed for the tab.<br>**Recommendation**: Although the maximum length is 64 characters, to ensure that the tab fits correctly in the ribbon, it's a good practice to limit the label to 16 characters.|
```
Comment (v-npaladugu) [inline msteams-platform/resources/schema/manifest-schema.md:1054]:
```suggestion
|`tabs.label`| String | 64 characters | | Specifies the text displayed for the tab. Despite the maximum length being 64 characters, for the tab to align correctly in the ribbon, we recommend you limit the label to 16 characters.|
```

File: msteams-platform/resources/schema/manifest-schema.md [hunk 5]
```diff
@@ -1050,14 +1050,14 @@ The `extensions.ribbons` property provides the ability to add [add-in commands](
...
 |`tabs.position.align`| String enum | | ✔️ | Defines the alignment of custom tab relative to the specified built-in tab. <br>Supported values: `after`, `before`|
-|`tabs.groups`| Array | 10 | | Defines groups of controls on a ribbon tab on a non-mobile device. For mobile devices, see `tabs.customMobileRibbonGroups` below.|
+|`tabs.groups`| Array | 10 | | Defines groups of controls on a ribbon tab on a non-mobile device. For mobile devices, see `tabs.customMobileRibbonGroups`.|
 |`tabs.groups.id`| String |64 characters | | Specifies the ID for the tab group within the app. It must be different from any built-in group ID in the Microsoft 365 application and any other custom group.|
-|`tabs.groups.label`| String | 64 characters | | Specifies the text displayed for the group. |
+|`tabs.groups.label`| String | 64 characters | | Specifies the text displayed for the group.<br>**Recommendation**: Although the maximum length is 64 characters, to ensure that the group of controls fits correctly in the ribbon, it's a good practice to limit the label to 16 characters.|
```
Comment (v-npaladugu) [inline msteams-platform/resources/schema/manifest-schema.md:1060]:
```suggestion
|`tabs.groups.label`| String | 64 characters | | Specifies the text displayed for the group. Despite the maximum length being 64 characters, for the group of controls to align correctly in the ribbon, we recommend you limit the label to 16 characters.|
```

File: msteams-platform/resources/schema/manifest-schema.md [hunk 6]
```diff
@@ -1078,7 +1078,7 @@ The `extensions.ribbons` property provides the ability to add [add-in commands](
...
 |`tabs.groups.controls.type`| String | | ✔️ | Defines the control type. <br>Supported values: `button`, `menu`|
 |`tabs.groups.controls.builtinControlId`| String | 64 characters | ✔️ | Specifies the ID of an existing Microsoft 365 control. For more information, see [find the IDs of controls and control groups](/office/dev/add-ins/design/built-in-button-integration#find-the-ids-of-controls-and-control-groups).|
-|`tabs.groups.controls.label`| String | 64 characters | ✔️ | Specifies the text displayed for the control.|
+|`tabs.groups.controls.label`| String | 64 characters | ✔️ | Specifies the text displayed for the control.<br>**Recommendation**: Although the maximum length is 64 characters, to ensure that the control fits correctly in the ribbon, it's a good practice to limit the label to 16 characters.|
```
Comment (v-npaladugu) [inline msteams-platform/resources/schema/manifest-schema.md:1081]:
```suggestion
|`tabs.groups.controls.label`| String | 64 characters | ✔️ | Specifies the text displayed for the control. Despite the maximum length being 64 characters, for the control to align correctly in the ribbon, we recommend you limit the label to 16 characters. |
```

**Review by v-npaladugu — CHANGES_REQUESTED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6c69ac44c6f81d857ef3fe65afe4733b72ca8bdf/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 84 | 126 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/cf4f5ea7-d05b-48a8-a7ed-079d84b0789c) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6c69ac44c6f81d857ef3fe65afe4733b72ca8bdf/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema.md) | 86 | 109 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/75dfd042-9528-4c96-a6ea-2000f7c05bba) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by Rick-Kirkham — APPROVED** [review]

**Review by Wajeed-msft — APPROVED** [review]

**Review by v-npaladugu — APPROVED** [review]

**Review by surbhigupta12 — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8cf4ecdaab40cf6f3e2b6a2325930748bc1b17c5/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 84 | 126 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2bde34fc-4cab-45ca-a0f7-62a180992eea) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8cf4ecdaab40cf6f3e2b6a2325930748bc1b17c5/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema.md) | 86 | 109 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2bafd183-0bee-48ce-a43e-4896b0f9fa85) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Pull request merged**